fnCompanions.sql

```diff
@@ -0,0 +1,29 @@
+CREATE FUNCTION fnCompanions(@EpisodeId INT)
+RETURNS VARCHAR(MAX) AS
+BEGIN
+	DECLARE @answer VARCHAR(MAX) = '';
```

Comment:
try something like this :
	declare @c varchar(100) = ''

	SELECT
		@c = @c + 
			CASE WHEN len(@c) > 0 THEN ', ' ELSE '' END + 
			c.CompanionName
			
	FROM
		tblEpisodeCompanion AS ec
		INNER JOIN tblCompanion AS c ON ec.CompanionId = c.CompanionId
	WHERE
		ec.EpisodeId = @EpisodeId